contract/r/gnoswap/protocol_fee/getter.gno

```diff
@@ -16,28 +14,58 @@ func TokenList(positive bool) []string {
 	return tokens
 }
 
+func GetDevOpsPct() uint64 {
+	return protocolFeeState.GetDevOpsPct()
+}
+
+func GetGovStakerPct() uint64 {
+	return protocolFeeState.GetGovStakerPct()
+}
+
 // GetTokenListWithAmount returns the token path and amount.
 func GetTokenListWithAmount() map[string]uint64 {
-	return protocolFeeState.TokenListWithAmount
+	return protocolFeeState.tokenListWithAmount
 }
 
 // GetAmountOfToken returns the amount of token.
 func GetAmountOfToken(tokenPath string) uint64 {
-	amount, exists := protocolFeeState.TokenListWithAmount[tokenPath]
+	amount, exists := protocolFeeState.tokenListWithAmount[tokenPath]
 	if !exists {
 		return 0
 	}
 	return amount
 }
 
-// GetAccuTransferToGovStaker returns the accuToGovStaker.
-func GetAccuTransferToGovStaker() *avl.Tree {
-	return protocolFeeState.AccuToGovStaker
+func GetAccuTransfersToGovStaker() map[string]int64 {
+	accuTransfers := make(map[string]int64)
+
+	protocolFeeState.accuToGovStaker.Iterate("", "", func(key string, value interface{}) bool {
```

Comment:
please use `any` type instead of `interface{}`
```suggestion
	protocolFeeState.accuToGovStaker.Iterate("", "", func(key string, value any) bool {
```